packages/@react-aria/overlays/src/useOverlayTrigger.ts

```diff
@@ -53,14 +53,21 @@ export function useOverlayTrigger(props: OverlayTriggerProps): OverlayTriggerAri
     };
   }, [isOpen, onClose, ref]);
 
+  // Aria 1.1 supports multiple values for aria-haspopup other than just menus.
+  // https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup
+  // However, we only add it for menus for now because screen readers often 
```

Comment:
this comment is now untrue?

Reply:
menus and listboxes are close enough where if it is exposed as a menu it's not that big of a deal. we did this in v2.